src/ui/onboard_view.rs

```diff
@@ -18,6 +18,8 @@ use crate::ui::{
 	},
 };
 
+use super::keyboard;
+
```

Comment:
we use `create` as import typically
```suggestion
use crate::keyboard::enter_variant_key_check;

```

But I guess I would rename the function to something like `is_enter` maybe?

Reply:
Oh also you should merge that import to the top of the file where the other crate imports are